From 4c66d54cf74bf2ebf7e1593d180411601508478b Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 30 Oct 2016 19:09:09 +0200
Subject: [PATCH 0550/1345] apn806: dts: add revision info node

commit  d5fffd1a1db29a190c6c2010cf98ab84c21e9263 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

A8k A0 requires workaround for SATA.
This node will be used by SATA driver to distinguish
A0 and A1 revisions.

Important notice:
- When A8k A0 becomes unsupported - This commit should be
  removed!
- Don't submit this patch upstream

Change-Id: I57a7a4b26a2b848634a79d0fb2c2774512e5b56c
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33478
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 085e823..134fe94 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -143,6 +143,11 @@
 				interrupts = <GIC_SPI 18 IRQ_TYPE_LEVEL_HIGH>;
 			};
 
+			revision-info@610fcc {
+				compatible = "marvell,ap806-rev-info";
+				reg = <0x610FCC 0x4>;
+			};
+
 			smmu: iommu@5000000 {
 				compatible = "arm,mmu-500";
 				reg = <0x100000 0x100000>;
-- 
1.7.9.5

